frontend/components/Spinner/_styles.scss

```diff
@@ -6,7 +6,9 @@
   animation-delay: 250ms;
   opacity: 0;
   &.centered {
-    margin: 120px auto;
```

Comment:
@ghernandez345 thanks, @RachelElysia also identified that issue above. I missed this because the `.centered` here is set using the `classnames` function (i.e., `classnames({centered})`), so when I global searched for `.centered` I didn't find anything, leading me to think `.centered` was only being used in this one place.